dev/com.ibm.ws.jaxrs.2.1_fat_extended/fat/src/com/ibm/ws/jaxrs21/fat/extended/ClassSubResTest.java

```diff
@@ -20,12 +20,13 @@
 import componenttest.annotation.SkipForRepeat;
 import componenttest.annotation.TestServlet;
 import componenttest.custom.junit.runner.FATRunner;
+import componenttest.rules.repeater.JakartaEE10Action;
 import componenttest.rules.repeater.JakartaEE9Action;
 import componenttest.topology.impl.LibertyServer;
 import componenttest.topology.utils.FATServletClient;
 import jaxrs21.fat.classSubRes.ClassSubResTestServlet;
 
-@SkipForRepeat(JakartaEE9Action.ID) // multiple errors needs investigation
+@SkipForRepeat({JakartaEE9Action.ID, JakartaEE10Action.ID}) // multiple errors needs investigation
```

Comment:
we should have an issue for this

Reply:
https://github.com/OpenLiberty/open-liberty/issues/22522